README.md

```diff
@@ -121,6 +121,7 @@ There are 2 extra required attributes:
 There are also two optional attributes:
 
 - `source_header`: Specify a particular email header to use to identify the source of a particular notification and assign it to the appropriate provider. If unset, `From` will be used, but if your emails are not received directly from the provider but instead pass through a mailing list or alias, you might need to set this to a different value such as `X-Original-Sender` instead.
+- `source_header_limit_emails`: Restrict the emails used as "From", maybe a email from a mailing list or alias, in order to limit the emails that are processed.
```

Comment:
This doesn't have anything to do with headers - perhaps "limit_to_emails_from" would be a more meaningful name?

Reply:
I wanted to make it clear that is related to the `source_header`, because the default way to limit emails is by the Providers custom field. 
Gonna try to get some better naming and also some test, I just created this PR quickly to get some feedback on the overall approach